packages/frontend/src/components/permissioned.tsx

```diff
@@ -20,12 +20,14 @@ export const Permissioned = ({ onBack }: PermissionedProps) => {
                     <>
                         <ErrorIcon className="w-52 h-52" />
                         <Typography
+                            data-testid="permission-mode-enabled-text"
```

Comment:
Should we rename this to `permission-mode-enabled-title-text`, to be more specific?

Reply:
agreed